kusto/trustedEndpoints/trustedEndpoints.go

```diff
@@ -0,0 +1,240 @@
+package truestedEndpoints
```

Comment:
file folder names - snake_case, not camelCase

Reply:
You didn't push? the folder name is still "trustedEndpoints" and not "trusted_endpoints"